lib/utils/useRemoteStorageMappings.js

```diff
@@ -1,13 +1,18 @@
 import { useQuery } from 'react-query';
 
-import { useOkapiKy } from '@folio/stripes-core';
+import { useOkapiKy, useStripes } from '@folio/stripes-core';
 
 const API = 'remote-storage/mappings';
 const LIMIT = 1000;
 
 const useRemoteStorageMappings = () => {
+  const stripes = useStripes();
   const ky = useOkapiKy();
 
+  const withRemoteStorage = (
+    stripes.hasInterface('remote-storage-configurations') && stripes.hasInterface('remote-storage-mappings')
+  );
+
   const searchParams = { limit: LIMIT };
   const queryKey = [API, searchParams];
   const queryFn = () => ky(API, { searchParams }).json();
```

Comment:
`const queryFn = () => ky(API, { searchParams }, enabled: withRemoteStorage ).json();`
 So if !withRemoteStorage you don't need to make request.